src/dbsync/include/dbsync.h

```diff
@@ -37,22 +37,28 @@
 #ifdef __cplusplus
 extern "C" {
 #endif
+/**
+ * @brief Initializes the shared library.
+ * @param log_function pointer to log function to be used by the dbsync.
```

Comment:
log_function should be changed to function in order to doxygen takes the correct name in the param